.github/workflows/CI.yaml

```diff
@@ -223,7 +223,7 @@ jobs:
           cargo test -p pallet-gear-payment --features=lazy-pages --release --locked
 
       - name: "Test: Gear workspace"
-        run: ./scripts/gear.sh test gear --exclude gclient --exclude gprogram --release --locked
+        run: ./scripts/gear.sh test gear --exclude gclient --exclude gprogram --features pallet-gear-debug/lazy-pages --release --locked
```

Comment:
I've made special pallets tests run with lazy pages feature, see above. Testing also without lazy-pages is a good possibility to found some unexpected problems. Also I cannot understand why only for `pallet-gear-debug` then?

Reply:
Sure, but: cargo is really lazy thing with it's own issues

1. Building workspace tests (by default, it contains lazy pages due to appearance in node with it by default)
2. pallet-gear-debug didn't fit any crate without features, which is disabled by default in workspace
3. pallet-gear-debug built wout lazy-pages and it thinks that pallet-gear goes same, but it built with lazy-pages => test fails, so it's easier to enable just lazy-pages feature

test passes if we call directly pallet-gear-debug with and without lazy-pages via `cargo test -p pallet-gear-debug`. It's only about workspace troubles

Reply:
Solved?